test/parallel/test-events-once.js

```diff
@@ -114,6 +119,81 @@ async function prioritizesEventEmitter() {
   process.nextTick(() => ee.emit('foo'));
   await once(ee, 'foo');
 }
+
+async function abortSignalBefore() {
```

Comment:
:thinking: Shouldn't we add `process.on('multipleResolves', common.mustNotCall())` for these tests?

Also, looking through this test https://github.com/nodejs/node/blob/b0e64bf48615acaeb48a8ea9ff529710ad55d527/test/parallel/test-events-once.js#L72 I found, why is it needed?

Reply:
I think multipleResolves is currently still broken until someone picks it up and fixes it so I don't think we should do that most likely